pages/index.tsx

```diff
@@ -56,8 +67,24 @@ export const HomePage: NextPage = () => {
       )
     }
 
+    if (selectedNeighborhoods.length > 0) {
+      const neighborhoodResources: string[] = []
+      selectedNeighborhoods.forEach(function (neighborhood: Neighborhood) {
+        if (neighborhood['Resources']) {
+          neighborhood['Resources'].forEach(function (resourceId: string) {
+            neighborhoodResources.push(resourceId)
+          })
+        }
+      })
+      filteredResources = filteredResources.filter(function (
+        resource: Resource
+      ) {
+        return neighborhoodResources.includes(resource['id'])
+      })
```

Comment:
If you make the changes suggested above, you should be able to simplify this to:

```
filteredResources = filteredResources.filter((resource) => resource.Neighbhorhoods && selectedResources.includes(resource.Neighborhoods))
```